src/__tests__/transformers/Streamable.js

```diff
@@ -0,0 +1,243 @@
+import cases from 'jest-in-case';
+import fetchMock from 'node-fetch';
+
+import plugin from '../../';
+import {
+  getHTML,
+  getNormalizedStreamableUrl,
+  shouldTransform,
+} from '../../transformers/Streamable';
+
+import { cache, getMarkdownASTForFile, parseASTToMarkdown } from '../helpers';
+
+jest.mock('node-fetch', () => jest.fn());
+
+const mockFetch = html =>
+  fetchMock.mockResolvedValue({ json: () => Promise.resolve({ html }) });
+
+beforeEach(() => {
+  fetchMock.mockClear();
+});
+
+cases(
+  'normalized streamable path validation',
+  ({ url, shortCode }) => {
+    expect(getNormalizedStreamableUrl(url)).toBe(shortCode);
```

Comment:
We should test the normalized url here, not the `shortCode`, because otherwise we're doing a call to `https://api.streamable.com/oembed.json?url=moo` and that's not valid